db/migrations/20221227080317_evaluation_mood.sql

```diff
@@ -0,0 +1,11 @@
+-- migrate:up
+CREATE TABLE evaluation_mood (
+  evaluation_id INT NOT NULL,
+  mood_id int NOT NULL,
+  FOREIGN KEY (evaluation_id) REFERENCES evaluations (id),
+  FOREIGN KEY (mood_id) REFERENCES moods (id)
+  );
```

Comment:
해당 테이블과 연관된 api를 구현할 때 id(pk)는 필요 없는게 맞나요? 필요 없다면 상관없겠지만 실수로 누락하신거라면 고민해볼 필요가 있을 것 같습니다.

또한 테이블간 M:N 관계를 정의하기 위한 중간 테이블 naming은 다음과 같이 수정해주세요
```suggestion
CREATE TABLE evaluations_moods (
  evaluation_id INT NOT NULL,
  mood_id int NOT NULL,
  FOREIGN KEY (evaluation_id) REFERENCES evaluations (id),
  FOREIGN KEY (mood_id) REFERENCES moods (id)
  );
```

Reply:
넵 id 값은 저희가 상의 하고 제외한 부분 입니다.